cypress/support/checkin.js

```diff
@@ -9,7 +10,7 @@ Cypress.Commands.add('checkInItem', barcode => {
   cy.do([
     TextField({ id: 'input-item-barcode' }).fillIn(barcode),
     Button('Enter').click(),
-    Button({ text: 'Check in', button: true }).click(),
+    Modal('Confirm multipiece check in').find(Button('Check in')).click(),
```

Comment:
Just a heads up, if you want to make the Modal match a little more flexible, you can use the `including()` matcher:

```js
Modal(including('Confirm')).find(Button('Check in')).click()
```

For more on matchers https://frontside.com/bigtest/docs/interactors/matchers#writing-flexible-assertions-using-matchers